pkg/lib/config/bot_protection.go

```diff
@@ -40,7 +40,7 @@ var _ = Schema.Add("BotProtectionProvider", `
 
 type BotProtectionConfig struct {
 	Enabled  *bool                  `json:"enabled,omitempty"`
-	Provider *BotProtectionProvider `json:"provider,omitempty" nullable:"true"`
+	Provider *BotProtectionProvider `json:"provider,omitempty"`
```

Comment:
Provider is nullable. And Please implement SetDefaults() on BotProtectionConfig to set Enabled to false if it is nil

Reply:
As per offline discussion, will make Enabled as non-pointer instead